src/Kmd.Logic.Audit.Client.SerilogLargeAuditEvents/SerilogLargeAuditEventClient.cs

```diff
@@ -14,20 +16,23 @@ public SerilogLargeAuditEventClient(SerilogLargeAuditEventClientConfiguration co
         {
             this.customSinkLogger = config.CreateDefaultAzureBlobOrEventHubConfiguration().CreateLogger();
             this.disposeLogger = true;
+            SelfLog.Enable(msg => Console.WriteLine(msg));
```

Comment:
I have included a log statement in `AzureBlobServiceProvider.cs`
`Serilog.Debugging.SelfLog.WriteLine($"Audit event with event id {eventId} of size {auditEventSize} bytes exceeding threshold, hence uploading blob in chunks.");`
This is just to log that blob upload is happening in chunks instead of direct upload. Hence used `Enable()` in `SerilogLargeAuditEventClient.cs`